src/main/java/homework3/ex2/page/Ex2DifferentElementsPage.java

```diff
@@ -0,0 +1,85 @@
+package homework3.ex2.page;
+
+import homework3.BasePage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class Ex2DifferentElementsPage extends BasePage {
+
+    @FindBy(css = "label:nth-child(1) > input[type=checkbox]")
```

Comment:
can be improved with xpath:
//label[contains(. ,'Wind')]/input


Improve other locators here too